replication/report/replication_report.go

```diff
@@ -162,3 +162,30 @@ func (f *FilesystemReport) NextStep() *StepReport {
 func (f *StepReport) IsIncremental() bool {
 	return f.Info.From != ""
 }
+
+// Returns, for the latest replication attempt,
+// 0  if there have not been any replication attempts,
```

Comment:
Is this a valid situation, a replication report with no attempts? If yes, I'd consider it to be wrong to not report an error, since nothing was replicated, or am I missing something here?
Also, 0 is returned if everything was successfully replicated.

Reply:
> Is this a valid situation, a replication report with no attempts? If yes, I'd consider it wrong to not report an error, since nothing was replicated, or am I missing something here?

When the daemon starts up, a push job will have 0 replication attempts reported until it is woken up via command line or because its snapshotter created a snapshot.